packages/mui-material/src/CardHeader/CardHeader.js

```diff
@@ -80,13 +80,14 @@ const CardHeader = React.forwardRef(function CardHeader(inProps, ref) {
   const {
     action,
     avatar,
-    className,
```

Comment:
`useSlot` already handle the `className` for root slot.
Confirmed by `testClassName` in `describeConformance()` suite.